.github/workflows/go.yml

```diff
@@ -57,6 +57,15 @@ jobs:
         with:
           args: --timeout=2m
 
+  YamlLint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: ⤵️ Check out code from GitHub
+        uses: actions/checkout@v2
+      - name: 🚀 Run yamllint
+        run: |
+          yamllint -f github -c .yamllint .
+
```

Comment:
@eloycoto I'd like to have your feedback on this. :)

Reply:
all good!